Accessories/uInverter/Design.md

```diff
@@ -0,0 +1,53 @@
+﻿## uInverter Design Calculations
+
+
+-   nominal dc link voltage - _12V_
+-   rated phase current - _2.5A rms_
+-   range of allowable switching frequency - _1kHz- 100kHz_
+-   range of allowable power frequency (phase frequency) - _640- 1000 Hz_
```

Comment:
Seems bizarre to limit the allowable phase current frequency to a minimum of 640Hz. I would presume that DC would be okay, meaning that this should be 0 to 1kHz range?